client/transport/grpc/src/main/java/io/a2a/client/transport/grpc/GrpcTransportConfigBuilder.java

```diff
@@ -0,0 +1,22 @@
+package io.a2a.client.transport.grpc;
+
+import io.a2a.client.transport.spi.ClientTransportConfigBuilder;
+import io.grpc.Channel;
+
+import java.util.function.Function;
+
+public class GrpcTransportConfigBuilder extends ClientTransportConfigBuilder<GrpcTransportConfig, GrpcTransportConfigBuilder> {
```

Comment:
Was just discussing these changes with some others and one thing I'm wondering is, do we really need builders for GrpcTransportConfig and JSONRPCTransportConfig since they each only have one thing that can be configured?

Reply:
That is a fair comment.

The current implementation is not strict with the use of a TransportBuilder, you can pass directly an instance of a TransportConfig when calling the `withTransport()` method.